addons/l10n_ch/models/res_bank.py

```diff
@@ -191,6 +192,11 @@ def _l10n_ch_get_qr_vals(self, amount, currency, debtor_partner, free_communicat
             # _check_for_qr_code_errors ensures we can't have a QR-IBAN without a QR-reference here
             reference_type = 'QRR'
             reference = structured_communication
+        # The structured communication can also be a valid creditor reference (ISO 11649)
+        # reference: https://github.com/arthurdejong/python-stdnum/blob/master/stdnum/iso11649.py
```

Comment:
Remove this link. We can see that you're calling stdnum from the code, and the line above is already making clear what you're checking exactly.